Tests/Functional/Controller/TeaControllerTest.php

```diff
@@ -22,6 +23,14 @@ final class TeaControllerTest extends FunctionalTestCase
 
     protected function setUp(): void
     {
+        ArrayUtility::mergeRecursiveWithOverrule($this->configurationToUseInTestInstance, [
+            'FE' => [
+                'cacheHash' => [
+                    'enforceValidation' => false,
```

Comment:
Could we also use a URL with a cHash in the test instead?

Reply:
Or maybe configure human-readable URLs in the site configuration?

Reply:
We do not have site configuration right now in functional test and we would also nee route enhancer for this.

Reply:
So would it be possible to create/use a valid cHash here?

Reply:
Done (without cHash, but with a simpler setup).